RedLock RQL command fix and update URL (#16520)
demisto/content: Packs/PrismaCloud/ReleaseNotes/2_0_5.md

#### Integrations
##### Prisma Cloud (RedLock)

> - Changed the default *Server URL* to: https://api.prismacloud.io/
> - Fixed an issue with **get_rql_response** command.
> - changed the default fetch time to 24 hours and fix calculation of fetch time.
> - Updated docker image to `demisto/python3:3.9.8.24399`

Reviewer: ```suggestion
- Changed the default *Server URL* to: https://api.prismacloud.io/
- Fixed an issue with **get_rql_response** command.
- Changed the default fetch time to 24 hours and fixed an issue where the first fetch would fetch all available alerts.
- Updated docker image to `demisto/python3:3.9.8.24399`
```
